bindings/python/CMakeLists.txt

```diff
@@ -1,3 +1,19 @@
+#
+# Copyright (c) 2017-2018 CNRS
+#
+# This file is part of tsid
+# pinocchio is free software: you can redistribute it
+# and/or modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation, either version
+# 3 of the License, or (at your option) any later version.
+# pinocchio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Lesser Public License for more details. You should have
+# received a copy of the GNU Lesser General Public License along with
+# pinocchio If not, see
```

Comment:
Here it still says "pinocchio" instead of "tsid"

Reply:
Thx. I modified the license statement in all CMakeList.